tests/templates/kuttl/smoke/50-install-druid.yaml.j2

```diff
@@ -47,6 +47,7 @@ spec:
     config:
       logging:
         enableVectorAgent: {{ lookup('env', 'VECTOR_AGGREGATOR') | length > 0 }}
+      resources:
```

Comment:
Intentional? I hope we don't need this change or?

Reply:
done